08-communicating-results.Rmd

```diff
@@ -215,7 +215,7 @@ The following summary statistics are available for continuous variables:
   - `{p##}` any integer percentile, where `##` is an integer from 0 to 100
```

Comment:
Do the ## mean that it has to be 2 digits? For example if we want the 5th percentile do we need to do p05?  That is what it looks like to me, and if it is not the case, then we should maybe change the ## to just be a single one?

Reply:
It is specified as p5 not p05 so I've changed to using 1 `#` mark.